src/services/ui/src/features/package-actions/lib/correctStatusSwitch.ts

```diff
@@ -0,0 +1,30 @@
+import { Action } from "shared-types";
+import {
+  idUpdated,
+  intakeCompleted,
+  packageWithdrawn,
+  raiIssued,
+  raiRespondedTo,
+  raiWithdrawalDisabled,
+  raiWithdrawalEnabled,
+  raiWithdrawn,
+  temporaryExtensionRequested,
+} from "./modules";
+import { CheckStatusFunction } from "@/utils/Poller/seaStatusPoller";
+
+export const correctStatusToStopPollingData = (a: Action) => {
```

Comment:
this is still hyperspecific to statuses but we can use this model to check _any_ data has updated. rename to something like `successCheckSwitch`, to mirror the `fieldSwitch`, `contentSwitch`, etc. model